test/sharness/t0180-private-network.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/sh
+#
+# Copyright (c) 2015 Jeromy Johnson
+# MIT Licensed; see the LICENSE file in this repository.
+#
+
+test_description="Test private network feature"
+
+. lib/test-lib.sh
+
+test_init_ipfs
+
+export LIBP2P_FORCE_PNET=1
```

Comment:
I'd  find very useful that there was a similar option in the ipfs config file. `"EnablePrivateNetworks"` kind of thing. Also, an option to indicate the path to the key?